inspire/modules/deposit/views.py

```diff
@@ -25,3 +42,73 @@
     template_folder='templates',
     static_folder="static",
 )
+
+
+CHART_TYPES = {'column': 'Column',
+               'pie': 'Pie'}
+
+
+class FilterDateForm(Form):
```

Comment:
Should we consider moving this form definition to another file? Just to avoid cluttering if we expand this file in the future? E.g. keeping blueprint only for entry points.